server.js

```diff
@@ -45,6 +45,10 @@ app.get('/search/:searchTerm', function(req, res, next) {
     next()
 })
 
+app.get('/make/', function(req, res, next) {
```

Comment:
Could you explain why you chose `make` for the path when the button is called create quiz?

Reply:
I just used it to check my work when formatting the web page

Reply:
Can we change it to `/create` please?